docs/howto/operate/add-aws-hub.md

```diff
@@ -0,0 +1,211 @@
+these environments vars for kops to use, so we do it manually).
+
+```bash
+export AWS_PROFILE=<cluster_name>
+export AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id)
+export AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key)
+```
+
+## Create an AWS kops-based cluster
+
+1. From the root directory on this repo, `cd` into the `kops` directory
+2. Set up a *state* bucket for kops. This bucket will store the cluster
+[state](https://kops.sigs.k8s.io/state/).
+
+``` bash
+export KOPS_STATE_STORE=s3://<2i2c>-<hub-name>-kops-state
+aws s3 mb $KOPS_STATE_STORE --region <region>
+```
+
+```{note}
+Kops [recommends](https://kops.sigs.k8s.io/getting_started/aws/#cluster-state-storage)
+versioning the S3 bucket in case you ever need to revert or recover a previous state
+store.
+```
+
+### Create and render a kops config file
+
+You can use 
+[one](https://github.com/2i2c-org/pilot-hubs/blob/master/kops/farallon.jsonnet) of the
+existing [jsonnet](https://jsonnet.org/) specifications as a "template" for your cluster.
+You may need to tweak zones, names and instances, the rest is boilerplate to create a
+kops-based cluster accordingly to some specification already outlined in 
+[#28](https://github.com/2i2c-org/pangeo-hubs/issues/28). Once you have your jsonnet
+specification ready, you need to render it to create the config file kops understand.
+
+1. Render the `kops` config file with
+
+```bash
+jsonnet <cluster_name>.jsonnet -y > <cluster_name>.kops.yaml
+```
+
+2. Regrettably, the rendering creates yaml file with with 3 dots at the end, you can 
+delete it with
+
+```bash
+sed -i '' -e '$ d' <cluster_name>.kops.yaml
+```
+
+```{note}
+In Linux you will need to use instead: `sed -i '$ d' <cluster_name>.kops.yaml`
+```
+
+### Create the cluster
+
+1. Create the cluster configuration and push it to s3 with
+
+```bash
+kops create -f <cluster_name>.kops.yaml
+```
+
+2. You will need to a ssh key pair before actually creating the cluster
+
+```bash
+ssh-keygen -f ssh-key
+mv ssh-key <cluster_name>.key
+mv ssh-key.pub <cluster_name>.key.pub
+```
+
+```{note}
+You will need to relocate and encrypt the private key before pushing your changes to the
+repository.
+```
+
+3. Build the cluster with (notice that you are passing the ssh public key you just 
+created in the previous step)
+
+```bash
+kops update cluster <cluster_name>hub.k8s.local --yes --ssh-public-key <cluster_name>.key.pub --admin
+```
+
+```{note}
+The `--admin` at the end will modify your `~/.kube/config` file to point to the new 
+cluster.
+```
+
+If everything went as expected, the cluster will be created after some minutes, and you 
+should be able to validate it with
+
+```bash
+kops validate cluster --wait 10m
+```
+
+But validation will not pass until this next section is done.
+
+### Apply workaround to run CoreDNS on the master node
+
+More details on this [issue](https://github.com/kubernetes/kops/issues/11199)
```

Comment:
````suggestion
```{seealso}
More details [in this issue](https://github.com/kubernetes/kops/issues/11199)
```
````

Reply:
Nice, thanks!